### ISSUE TITLE
failed to simulate clients: workspace/[...]/testInfo.json: no such file or directory

### ISSUE THREAD

**Issue by whilei** [ethereum/hive#175]
```
$ uname -a
Linux ubp52 4.17.4-041704-generic #201807031235 SMP Tue Jul 3 12:37:29 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
$ git rev-parse HEAD
afe29a12627e1d2f901d50502bb1c65cae5342ea
```

```
~/go/src/github.com/ethereum/hive master=
$ go install . && sudo $(which hive) --docker-noshell --client=go-ethereum:local --sim=devp2p.* --test=NONE --loglevel 10 --sim-rootcontext
INFO[02-13|13:59:21] docker daemon online                     version=18.06.1-ce
INFO[02-13|13:59:21] building clients for validation          pattern=go-ethereum:local
INFO[02-13|13:59:21] building validators for testing          pattern=NONE
INFO[02-13|13:59:21] creating ethash container 
INFO[02-13|13:59:21] building new docker image                nocache=false
Step 1/3 : FROM ethereum/client-go
 ---> 2173ad694620
Step 2/3 : RUN   echo '#!/bin/sh'                          > /root/ethash.sh &&   echo 'set -e'                            >> /root/ethash.sh &&   echo 'geth makedag 1 /root/.ethash'      >> /root/ethash.sh &&   echo 'if [ "$UID" != "0" ]; then'        >> /root/ethash.sh &&   echo '  adduser -u $UID -D ethash'       >> /root/ethash.sh &&   echo '  chown -R ethash /root/.ethash/*' >> /root/ethash.sh &&   echo 'fi'                                >> /root/ethash.sh &&   chmod +x /root/ethash.sh
 ---> Using cache
 ---> dc6ce8b0d6b7
Step 3/3 : ENTRYPOINT ["/root/ethash.sh"]
 ---> Using cache
 ---> 3762ee8a0502
Successfully built 3762ee8a0502
Successfully tagged hive/internal/ethash:latest
DBUG[02-13|13:59:21] created ethash container 
INFO[02-13|13:59:21] generating genesis DAG 
DBUG[02-13|13:59:21] attaching to container 
DBUG[02-13|13:59:21] starting container 
DBUG[02-13|13:59:22] deleting ethash container 
INFO[02-13|13:59:22] building clients for simulation          pattern=go-ethereum:local
INFO[02-13|13:59:22] building simulators for testing          pattern=devp2p.*
INFO[02-13|13:59:22] building new docker image                simulator=devp2p/basic nocache=false
Step 1/18 : FROM golang:1-alpine
 ---> cb1c8647572c
Step 2/18 : RUN apk add --update git make gcc musl-dev curl jq linux-headers
 ---> Using cache
 ---> 2392692947ce
Step 3/18 : RUN (git clone -b master --single-branch https://github.com/ethereum/go-ethereum /go/src/github.com/ethereum/go-ethereum)
 ---> Using cache
 ---> dd7cc66a5232
Step 4/18 : RUN (cd /go/src/github.com/ethereum/go-ethereum )
 ---> Using cache
 ---> 99e4fcbfb869
Step 5/18 : RUN (cd /go/src/github.com/ethereum/go-ethereum && GOPATH=/go make all)
 ---> Using cache
 ---> 8049e9e5da6d
Step 6/18 : RUN (cd /go/src/github.com/ethereum/go-ethereum && GOPATH=/go go get)
 ---> Using cache
 ---> 9c7004afda11
Step 7/18 : RUN (git clone -b master https://github.com/fsouza/go-dockerclient /go/src/github.com/fsouza/go-dockerclient)
 ---> Using cache
 ---> 858915e009c0
Step 8/18 : RUN (cd /go/src/github.com/fsouza/go-dockerclient && GOPATH=/go go get)
 ---> Using cache
 ---> d4b9de2fe6c9
Step 9/18 : RUN go get -u -v github.com/derekparker/delve/cmd/dlv
 ---> Using cache
 ---> 26748f67e674
Step 10/18 : ADD /devp2p/basic/devp2p_test.go /go/src/github.com/ethereum/hive/simulators/devp2p/basic/devp2p_test.go
 ---> Using cache
 ---> 2641397ccd16
Step 11/18 : ADD /devp2p/node.go  /go/src/github.com/ethereum/hive/simulators/devp2p/node.go
 ---> Using cache
 ---> debb27148539
Step 12/18 : ADD /devp2p/udp.go  /go/src/github.com/ethereum/hive/simulators/devp2p/udp.go
 ---> Using cache
 ---> 58e466b180f2
Step 13/18 : ADD common /go/src/github.com/ethereum/hive/simulators/common
 ---> Using cache
 ---> e3ae8079598b
Step 14/18 : RUN apk add --update bash curl jq
 ---> Using cache
 ---> d4cb0a2935ab
Step 15/18 : ADD /devp2p/basic/tests.sh /tests.sh
 ---> d8aff4835989
Step 16/18 : ADD /devp2p/basic/genesis.json /genesis.json
 ---> af06305c905a
Step 17/18 : EXPOSE 2345 8545 8546 8080 30303 30303/udp
 ---> Running in c1eacf072d74
Removing intermediate container c1eacf072d74
 ---> 5b1cf27ca407
Step 18/18 : ENTRYPOINT ["/tests.sh"]
 ---> Running in 413af2cb8c38
Removing intermediate container 413af2cb8c38
 ---> 6e9fff1eaab4
Successfully built 6e9fff1eaab4
Successfully tagged hive/simulators/devp2p/basic:latest
CRIT[02-13|13:59:23] failed to simulate clients               error="open workspace/logs/20190213135921/simulator_devp2p_basic/testInfo.json: no such file or directory"
```

**Comment by whilei** [comment]
So I think this is actually b/c of a misconfigured or badly parsed client list, and the fix in 176 is probably solving the symptom and not the cause. Going to close that one and dig a little deeper.

**Comment by adilimroz** [comment]
Hello @whilei , Did you by any chance get anything on this ? I am getting same issue with parity client and am trying to find a way to resolve this.

**Comment by whilei** [comment]
@adilimroz It's an issue with parsing the --client name. I fixed this issue w/ #179. 

You used 

```
 --client=parity:stable --test=none --sim=ethereum/consensus --loglevel=6
```

Instead, try

```
 --client='parity.*stable' --test=none --sim=ethereum/consensus --loglevel=6
```

**Comment by adilimroz** [comment]
Thanks @whilei  :)
